docs/Installing.asciidoc

```diff
@@ -633,6 +633,16 @@ The openQA web UI should be available on http://localhost/ now. To simply
 start openQA without enabling it permanently one can simply use `systemctl
 start` instead.
 
+== Enable automatic database backup
+An optional systemd service, `openqa-dump-db.service`, can be enabled to perform daily database backups. This service is triggered by the `openqa-dump-db.timer`.
```

Comment:
Please break this line at a similar length as it is done in surrounding paragraphs.

Reply:
Done